File: Python/List-2/has22.py
def has22(nums):
  b = False
  if len(nums) == 2:
    return nums[0] == nums[1] 
  for i in range(1,len(nums)-1):
    if (nums[i] == 2 and nums[i-1] == 2) or (nums[i] == 2 and nums[i+1] == 2):
      b = True
    if b:
      return b
  return b

File: Java/Array-2/has22.java
public boolean has22(int[] nums) {
  boolean output = false;
  for(int i = 0; i < nums.length; i++){
    if(nums[i] == 2 && i > 0 && nums[i-1] == 2){
        output = true;
    }
    else if(nums[i] == 2 && i < nums.length -1 && nums[i +1] == 2){
      output = true;
    }
  }
  return output;
}
